### PR TITLE
Hello microverse project

### PR DESCRIPTION
**Pull request by KayLemba**
1. Add index.html file that includes a <header> tag with the text "Hello Microverse!" 
2. Add styles.css file and change colour of <h1> tag.
3. Add test.md file with text "This file should be ignored by git".
4. Add test.md to gitignore.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (carlos-ssh) [inline .gitignore]: - Keep your project professional add into your gitignore external files like: .DS_Store, .idea or node_modules.
(In your root directory, you have an DS_Store file)

File: README.md [hunk 2]
```diff
@@ -1,22 +1,20 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (carlos-ssh) [inline README.md:1]: - Kindly fill your README template with the information of your project. 
(Write a descriptive and easy to understand README in business English for every project submitted.)
- Add information for non-technical people on how to download your repo and how to install and run it locally.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @KayLemba , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (KayLemba) [reply .gitignore]: @carlos-ssh Let me make those changes right away. Thank you.

**Review by mke2111 — APPROVED** [review]
### Hi @KayLemba, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**